pallets/pallet-dyn-filter/Cargo.toml

```
@@ -0,0 +1,63 @@
+[package]
+authors = ["KILT <info@kilt.io>"]
+description = "Dynamically filter calls."
+edition = "2021"
+name = "pallet-dyn-filter"
+repository = "https://github.com/KILTprotocol/kilt-node"
+version = "1.7.1"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dev-dependencies]
+itertools = "0.10.3"
+lazy_static = "1.4.0"
+pallet-balances = {branch = "polkadot-v0.9.27", default-features = false, git = "https://github.com/paritytech/substrate"}
+sp-core = {branch = "polkadot-v0.9.27", git = "https://github.com/paritytech/substrate"}
+sp-io = {branch = "polkadot-v0.9.27", git = "https://github.com/paritytech/substrate"}
+sp-keystore = {branch = "polkadot-v0.9.27", git = "https://github.com/paritytech/substrate"}
+
+[dependencies]
+codec = {package = "parity-scale-codec", version = "3.1.5", default-features = false, features = ["derive"]}
+kilt-support = {default-features = false, path = "../../support"}
+log = "0.4.17"
+scale-info = {version = "2.1.1", default-features = false, features = ["derive"]}
+
+# Substrate dependencies
+frame-support = {branch = "polkadot-v0.9.27", default-features = false, git = "https://github.com/paritytech/substrate"}
+frame-system = {branch = "polkadot-v0.9.27", default-features = false, git = "https://github.com/paritytech/substrate"}
+sp-core = {branch = "polkadot-v0.9.27", default-features = false, git = "https://github.com/paritytech/substrate"}
+sp-io = {branch = "polkadot-v0.9.27", default-features = false, git = "https://github.com/paritytech/substrate"}
+sp-runtime = {branch = "polkadot-v0.9.27", default-features = false, git = "https://github.com/paritytech/substrate"}
+sp-std = {branch = "polkadot-v0.9.27", default-features = false, git = "https://github.com/paritytech/substrate"}
+
+# benchmarking
+frame-benchmarking = {branch = "polkadot-v0.9.27", optional = true, default-features = false, git = "https://github.com/paritytech/substrate"}
+
+[features]
+default = ["std"]
+
+runtime-benchmarks = [
+  "frame-benchmarking",
+  "frame-support/runtime-benchmarks",
+  "frame-system/runtime-benchmarks",
+  "kilt-support/runtime-benchmarks",
+]
+
+std = [
+  "codec/std",
+  "frame-benchmarking/std",
+  "frame-support/std",
+  "frame-system/std",
+  "log/std",
+  "scale-info/std",
+  "sp-core/std",
+  "sp-io/std",
+  "sp-runtime/std",
+  "sp-std/std",
+  "kilt-support/std",
```

Not lexicographically ordered
```suggestion
  "codec/std",
  "frame-benchmarking/std",
  "frame-support/std",
  "frame-system/std",
  "kilt-support/std",
  "log/std",
  "scale-info/std",
  "sp-core/std",
  "sp-io/std",
  "sp-runtime/std",
  "sp-std/std",
```

---

[7d2ddc0](https://github.com/KILTprotocol/kilt-node/pull/397/commits/7d2ddc05ea3d21a15a0f766444934b2c8c37ac97)